Auth0/_ObjectiveWebAuth.swift

```diff
@@ -129,7 +129,7 @@ public class _ObjectiveOAuth2: NSObject {
      */
     @objc(resumeAuthWithURL:options:)
     public static func resume(_ url: URL, options: [A0URLOptionsKey: Any]) -> Bool {
-        return TransactionStore.shared.resume(url, options: options)
```

Comment:
why remove options?

Reply:
Because it’s not used at all, and simplifies matters if the transaction store has only one method for all platforms

Reply:
@cocojoe if you're happy with the changes, please bless the PR ✨